### PR TITLE
[ISSUE #3121] remove the disabled plugin from webhandler's plugins

### LINKED ISSUE #3121: [Task] Optimize  plugin configuration
### Description

Optimize admin plugin configuration and sync plugin state to ShenYu

### Task List

1. sync admin plugin configuration state to Shenyu bootstrap
2. remove the disabled plugin from webhandler's `plugins`

### PR DESCRIPTION
**Pull request by moremind**
// Describe your PR here; eg. Fixes #3121 

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

**Comment by moremind** [comment]
I think if remove plugin, maybe open local dispatcher and change AbstractPluginDataInit HttpHelper of shenyu-integrated-test,otherwise  this PR can't check successfully.

**Comment by moremind** [comment]
I can't find bug of integrated test,can you help me? @yu199195

File: shenyu-web/src/main/java/org/apache/shenyu/web/handler/ShenyuWebHandler.java [hunk 1]
```diff
@@ -54,6 +59,22 @@ public final class ShenyuWebHandler implements WebHandler, ApplicationListener<S
...
 
+    /**
+     * source plugins, these plugins load from ShenyuPlugin.
+     */
+    private final List<ShenyuPlugin> sourcePlugins;
+
+    /**
+     * default enabled plugins.
+     */
+    private final Set<String> defaultEnabledPlugins = Sets.newHashSet(PluginEnum.GLOBAL.getName(),
```
Comment (yu199195) [inline shenyu-web/src/main/java/org/apache/shenyu/web/handler/ShenyuWebHandler.java:70]: its error，remove this

Comment (moremind) [reply shenyu-web/src/main/java/org/apache/shenyu/web/handler/ShenyuWebHandler.java:70]: if remove it, default plugin not work.

**Review by yu199195 — APPROVED** [review]

**Pull request merged**